sourcecode/apis/contentauthor/app/Console/Commands/H5PLibraryInstall.php

```diff
@@ -0,0 +1,122 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Console\Commands;
+
+use App\Console\Libraries\CliH5pAjax;
+use App\H5PLibrariesHubCache;
+use App\H5POption;
+use Carbon\Carbon;
+use Illuminate\Console\Command;
+
+class H5PLibraryInstall extends Command
+{
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'h5p:library-manage
+                                {action   : install, update-cache}
```

Comment:
Separate actions ought to be separate commands so they can be listed when running `php artisan`. There's no way of intuitively knowing which actions are available without running `artisan` again with `--help`. But it'd be better if there were an `--update-cache` flag which updates the cache before installing in a single run instead.